packages/web-components/src/components/va-pagination/va-pagination.scss

```diff
@@ -55,3 +56,13 @@ button {
   transition: all 0.3s ease-in-out;
   transition-property: color, background-color;
 }
+
+@container (max-width: 480px) {
+  // This value matches the USWDS --mobile-lg custom property
+  // See https://designsystem.digital.gov/design-tokens/spacing-units/#spacing-unit-tokens
```

Comment:
Can we link to the VADS documentation instead in this comment? https://design.va.gov/foundation/breakpoints